languages/common-lisp/exercises/concept/arithmetic/.docs/introduction.md

```diff
@@ -0,0 +1,37 @@
+## Numerical Types
+
+In Common Lisp, like many languages, numbers come in a couple of types. This
+exercise focuses on getting to grips with two of the most basic number-types:
```

Comment:
The `introduction.md` is usually written as just being about the Concept(s) involved, without any reference to the exercise. It is not wrong per se, but maybe do:

```suggestion
In Common Lisp, like many languages, numbers come in a couple of types. The two most basic number-types are:
```